### PR TITLE
Pulsar: add bytes counters

### PR DESCRIPTION
**Pull request by eolivelli**
- Add "bytes" counter in order to track the number for bytes written to Pulsar and see the thoughtput in bytes and not in ops/sec.
- Fix dependencies

### CODE REVIEW

**Comment by eolivelli** [comment]
@jshook is this the right way to add a counter to see the "thoughtput" ? or should I use some other kind of metric ?

File: driver-pulsar/pom.xml [hunk 1]
```diff
@@ -27,7 +27,7 @@
 
         <dependency>
             <groupId>org.apache.pulsar</groupId>
-            <artifactId>pulsar-client</artifactId>
+            <artifactId>pulsar-client-original</artifactId>
```
Comment (jshook) [inline driver-pulsar/pom.xml:30]: What does this change do?

Comment (eolivelli) [reply driver-pulsar/pom.xml:30]: "pulsar-client" is a JAR that contains all of the dependencies of Pulsar, shaded, it is like an uber jar.

Comment (eolivelli) [reply driver-pulsar/pom.xml:30]: without this change IntelliJ messes up.

I didn't check how we package drivers. is it a problem if this jar adds its own third party deps to the NB build ?

Comment (eolivelli) [reply driver-pulsar/pom.xml:30]: @jshook I can revert these changes to pom.xml

**Comment by eolivelli** [comment]
by the way I cannot see this new metric after the execution of NB.
is there is something more to do in order to see the metric published into the CSV files ?

File: driver-pulsar/pom.xml (hunk 1 above)
Comment (jshook) [reply driver-pulsar/pom.xml:30]: The artifacts need to be resolvable in a public repository or it will break the build for everybody.

**Comment by jshook** [comment]
> @jshook is this the right way to add a counter to see the "thoughtput" ? or should I use some other kind of metric ?

I think it is suitable for tracking the cumulative payload size sent, but it will not give you any direct insights of payloads over time without doing your own derivatives, nor will it provide histograms.
You could also use a histogram if you want more texture of the data over time. It might make sense to keep the counter and a histogram both, since the counter is lightweight and provides a baseline signal as is.

File: driver-pulsar/pom.xml (hunk 1 above)
Comment (jshook) [reply driver-pulsar/pom.xml]: I see that this is also a public artifact. No issues on this from me.

**Review by jshook — APPROVED** [review]
This look ok. It might be useful to hold these metric objects in the ready op at some point.

**Pull request merged**